plugins/com.google.cloud.tools.eclipse.preferences/src/com/google/cloud/tools/eclipse/preferences/areas/PreferenceArea.java

```diff
@@ -25,11 +25,11 @@
 import org.eclipse.ui.IWorkbench;
 
 /**
- * Abstract base class for preference areas. A preference area is somewhat like a
- * {@link org.eclipse.jface.preference.FieldEditor FieldEditor} except that it is expected to be
- * configured from an extension point.
- * <p>
- * Preference areas should notify its listeners of changes using property change events.
+ * Abstract base class for preference areas. A preference area is somewhat like a {@link
```

Comment:
a --> an

Reply:
In the text it will appear as "like a Field Editor"